lib/Mitigation.pm

```diff
@@ -330,7 +330,8 @@ sub check_sysfs {
                     update_tss_attr(file_name => "$junit_file", attr => 'failures', value => $total_failure_tc_count);
                     update_tss_attr(file_name => "$junit_file", attr => 'tests', value => $total_tc_count);
                     parse_junit_log("$junit_file");
-                    die "Fail on the $self->{sysfs}->{$value}->{$sysfs_name_item}"; }
```

Comment:
Why need to remove "die", which will cause the program terminate when case is failed. is what you want?

Reply:
Yes.I don't want to terminate the program once meeting failed case.So I remove die.
I want all the cases about mitigations can be executed.

Reply:
OK, LGTM